make/conf/jib-profiles.js

```diff
@@ -1030,7 +1046,7 @@ var getJibProfilesDependencies = function (input, common) {
 
     var devkit_platform_revisions = {
         linux_x64: "gcc10.2.0-OL6.4+1.0",
-        macosx_x64: "Xcode11.3.1-MacOSX10.15+1.1",
+        macosx: "Xcode12.4+1.0",
```

Comment:
XCode bump w/o check for aarch?

Reply:
This seems to be used at line 1133 where 11.3.1 is hard-coded for x64. So I guess it works but I'm not very fond of this solution. @vidmik Why don't you put both x64 and aarch64 revisions here?

Reply:
As I understand it, the intention is to bump the Xcode version for x64 as well, though I must say I would prefer that change to go in separately.

Reply:
To answer Magnus, on line 1133, we define a different dependency, called "lldb". That dependency is an older devkit which only purpose is to provide lldb for the failure handler when running tests. We need this because we are still running tests on Macos < 10.15, but the new devkit requires 10.15.

Reply:
I agree that it would be ideal for the update to Xcode 12.x to be done separately. Is there a JBS issue filed for this update? Alternatively, you could use `/issue add nnnnnnn` to add that JBS issue to this PR.

Reply:
Thank you for catching the accidental Xcode update for macosx-x64. It was not my intention to include that as part of this change.

That said, right now it's looking highly likely that we will have bumped it for macosx-x64 before this PR is ready for integration anyway. Therefore, for now I will keep it as-is, but integration of this PR is effectively blocked until it has been resolved - either by making it conditional on the architecture or by the switch to "Xcode12.4+1.0" for macosx-x64 making it into mainline.

As Erik mentions, the new "lldb" dependency is needed to support running the failure handler+lldb when testing on older macOS versions which do not support Xcode 12.4. Assuming the switch to Xcode 12.4 as devkit for macosx-x64 makes it into mainline I believe the logic is correct.